pom.xml

```diff
@@ -201,6 +201,10 @@
     <version.net.byte-buddy>1.4.16</version.net.byte-buddy>
     <version.commons-beanutils>1.9.2</version.commons-beanutils>
     <version.docker-client>3.5.12</version.docker-client>
+    <version.gson>2.6.2</version.gson>
```

Comment:
@wmedvede I have removed the gson dependency and now just use what is in jboss-integration-platform-parent.